webpack/assets/javascripts/foreman_http_proxies.js

```diff
@@ -0,0 +1,26 @@
+/* global tfm*/
+/* eslint no-undef: "error"*/
```

Comment:
import $ from 'jquery';
import { notify } from './foreman_toast_notifications';

Reply:
added!  thanks for the tips. have not used webpack before.

Reply:
I think it's safe now to remove these comments.